packages/plugin-core/assets/dendron-ws/tutorial/tutorial.4-images-and-diagrams.md

```diff
@@ -0,0 +1,78 @@
+---
+id: vAy0awBTwDaxKGHWUdSaP
+title: Images and Diagrams
+desc: ''
+updated: 1624452350303
+created: 1624352425213
+---
+After completing this section you'll know how to create rich content within your Markdown notes.
+
+### Images
+
+As you may have already noticed already in the tutorial pages, you can insert images into your notes. 
+
+> 🌱 The easiest way to do this is to first download the [Paste Image](https://marketplace.visualstudio.com/items?itemName=mushan.vscode-paste-image) extension. Once installed, copy the image onto your clipboard, and then use the `Paste Image` command into your editor pane.
+
+This will automatically create a link for you and copy the file contents into the `\assets\images\` directory in your workspace.
+
+You can take advantage of the rich ecosystem of extensions available to VS Code to enhance your note taking abilities within Dendron.
+
+You can also manually place an image in the assets folder and create the link using the Markdown format below.
+
+Sample Image Link: ![](assets\images\logo.png)
```

Comment:
so here's a thing - we don't support multi-os image paths yet. if you hardcode backslashes here, it will fail in linux devices... 

you might just want to ask people to run paste images instead of adding an example

Reply:
Changed to forward slashes - seems to work on Windows as well:
`![Dendron Logo](assets/images/logo_small.png)`